lib/money-rails/active_model/validator.rb

```diff
@@ -41,6 +41,11 @@ def validate_each(record, attr, _value)
           raw_value = raw_value.to_s.gsub(symbol, "")
         end
 
+        if raw_value.present? && raw_value.is_a?(Hash)
+          
```

Comment:
this is rather pedantic, but would you please remove this unneeded empty line

Reply:
Heh, no problem :smile: